cmd/kube-scheduler/app/options/options.go

```diff
@@ -154,6 +157,15 @@ func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 	leaderelectionconfig.BindFlags(&o.ComponentConfig.LeaderElection.LeaderElectionConfiguration, nfs.FlagSet("leader election"))
 	utilfeature.DefaultMutableFeatureGate.AddFlag(nfs.FlagSet("feature gate"))
 
+	// TODO(RainbowMango): move it to genericoptions before next flag comes.
```

Comment:
It seems no generic options in the scheduler.
```suggestion
```

Reply:
I would still expect that we will refactor it to some options in component-base/metrics like https://github.com/kubernetes/kubernetes/pull/85266